src/resources/views/crud/edit.blade.php

```diff
@@ -57,9 +57,9 @@
 		    @endif
 		      <!-- load the view from the application if it exists, otherwise load the one in the package -->
 		      @if(view()->exists('vendor.backpack.crud.form_content'))
-		      	@include('vendor.backpack.crud.form_content', ['fields' => $crud->fields(), 'action' => 'edit'])
+		      	@include('vendor.backpack.crud.form_content', ['action' => 'edit'])
 		      @else
-		      	@include('crud::form_content', ['fields' => $crud->fields(), 'action' => 'edit'])
```

Comment:
There is no harm in passing here the `$crud->fields()` but is also not needed.

Reply:
@tabacitu sure about that, but we already have calls to the crud object after `form_content` in the chain. 

Tabbed fields are included in form content: https://github.com/Laravel-Backpack/CRUD/blob/master/src/resources/views/crud/inc/show_tabbed_fields.blade.php 

Should we address that ?

Also there is other perspective to the multiple form problem, is moving that logic to the singleton, so make a crud panel have a `forms` property and make the crud panel "multi-form", that way when you pass `$crud` to the form, you already pass the `$crud` with a predefined form selected, so all operations would happen to that form get fields etc. 

Let me know,
Pedro

Reply:
Hmm you're right. We have _other_ calls to `$crud`... 

I still think we should calculate fields at the top here, rather than each one individually. I'd pass `['fields'= > $crud->fieldsWithOverwrittenNames()]` to `form_content.blade.php`, if we do https://github.com/Laravel-Backpack/CRUD/pull/3981#discussion_r761805530